matopeli-beta.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Snake Game</title>
```

Comment:
Varmaan pitäis suomeksi olla tämä?

Reply:
Kyllä vaihdetaan tämä suomeksi. Pitäisin myös titlenä "Ota yhteyttä", mikäli matopeli liitetään ko. sivulle. Otsikko muotoon sitten "Matopeli".

Reply:
Ei muuta huomioita mullakaan kun mitä Niklas huomasi.

Reply:
Asia kunnossa.